tests/sentry/incidents/test_models.py

```diff
@@ -167,11 +167,11 @@ class AlertRuleTriggerActionActivateTest(object):
     method = None
 
     def setUp(self):
-        self.old_handlers = AlertRuleTriggerAction.handlers
-        AlertRuleTriggerAction.handlers = {}
+        self.old_handlers = AlertRuleTriggerAction._type_registrations
+        AlertRuleTriggerAction._type_registrations = {}
```

Comment:
Should there be public methods for these operations?

Reply:
I don't know that we should. It's gross to dive into variables that are basically private, but in practice I only want to do this in tests and don't really want a public interface for people to change these.